packages/graphql/package.json

```diff
@@ -16,7 +16,8 @@
   "browser": "dom.js",
   "files": [
     "node.js",
-    "dom.js"
+    "dom.js",
+    "lib"
```

Comment:
Should we maybe switch to using `.npmignore` files instead to not run into the same issue again?

Reply:
Sounds good, but I wouldn't want to do that just now.